README.md

```diff
@@ -45,6 +45,51 @@ that aren't ) are directly accessible for use writing your own custom
 research tools...  The interface should be nearly the same when dealing with
 a real process ( via vdb/vtrace ) and dealing with an emulator / viv workspace.
 
+## UI Dependencies
```

Comment:
great work!  

however, since we *just* wrapped in PyQt5 support, do you mind testing that and adding anything you think would be helpful getting PyQt5 environment up and running?